.github/workflows/release_latest.yml

```diff
@@ -18,13 +21,26 @@ jobs:
         with:
           # Must use at least depth 2!
           fetch-depth: 2
-      - uses: actions/setup-python@v2
+
+      - name: Setup python3
```

Comment:
seems that use `actions/setup-python@v2` to install python in this container will meet some troubles,  will get error when use pip3 to install awscli, don't know why,
so just install it by yum.